connector.rb

```diff
@@ -1,37 +1,2798 @@
 {
-  title: 'Template Partner Repo',
+  title: '4me',
+  secure_tunnel: false,
 
   connection: {
+    fields: [
+      {
+        name: 'account',
+        label: '4me Account',
+        optional: false,
```

Comment:
`false` is default value for optional

Reply:
When using the online code editor and debug console I have to explicitly specify it, otherwise, the fields are not required in the debug console